SteamKit2/SteamKit2/Steam/CDN/Client.cs

```diff
@@ -230,18 +229,26 @@ public async Task<int> DownloadDepotChunkAsync( uint depotId, DepotManifest.Chun
             var chunkID = Utils.EncodeHexString( chunk.ChunkID );
             var url = $"depot/{depotId}/chunk/{chunkID}";
 
-            using var request = new HttpRequestMessage( HttpMethod.Get, BuildCommand( server, url, cdnAuthToken, proxyServer ) );
-
+            var builtUrl = BuildCommand( server, url, cdnAuthToken, proxyServer );
+            var request = new HttpRequestMessage( HttpMethod.Get, builtUrl );
+            if ( UseLancacheServer )
```

Comment:
the two calls above shouldn't happen for lancache then, just being overriden.

Reply:
Split it out into an if-else